### ISSUE TITLE
Responsive issue

### ISSUE THREAD

**Issue by aajinkya1203** [casbin/casbin-website#126]
There seems to be a responsive issue for some devices like:

- [ ] iPhone 5/SE
- [ ] Galaxy S5

Uploading an image for reference!
![image](https://user-images.githubusercontent.com/43881544/92562725-0f0fd980-f294-11ea-90a9-41d7b5074fed.png)

**Comment by aajinkya1203** [comment]
This was done using the Chrome Dev Tools, if it helps any

**Comment by nodece** [comment]
My browser is work.

## ENV
- MacOS 10.15.5 
- Chrome 85.0.4183.102（64 bit）

**Comment by hsluoyz** [comment]
@aajinkya1203 can you fix it? Can you see if this site: https://docusaurus.io/ also has the same problem?

**Comment by aajinkya1203** [comment]
Hey, @hsluoyz 
I went through the website and it works fine!
I noticed that the error still persists but after I reloaded my site, it seemed to take the full width

My takeaway:
- [ ] The website works fine on phone
- [ ] Could be a litte problem on browsers if someone shrinks their browsers, although no one ever does it

I'll try from other devices too and would let you know. Closing this because as of now reloading does the work so I'm guessing its got something to do with the Chrome Dev Tools.
